README.rst

```diff
@@ -82,6 +82,20 @@ at creation time, so that this value will be used by default.
     client = Client(bucket="payments", collection="receipts", auth=auth)
 
 
+Getting server information
+--------------------------
+
+You can use the ``server_info`` methods to get the server information::
+
+.. code-block:: python
+
+    from kinto_client import Client
+
+    client = Client(server_url='http://localhost:8888/v1')
+    infos = client.server_info()
```

Comment:
I would say it's a "mass noun" like "rice" and can't be either singular or plural. I know that this is different in French where you can say "une information" -- I would say "a piece of information" or "some information".